perception/mil_mlp/README.md

```diff
@@ -0,0 +1,168 @@
+# Transfer Learning Object Detection - A Short Tutorial
+
+# @TODO
+1. Configurable image download
+2. Add docker build to the install script. Until then docker build must be done manually.
+3. Have datasets stored on internal server so it can be downloaded.
+4. Configurable resize (Low importance due to no significant performance gain)
+5. Configurable CSV input (Currently it still assumes you are handing it a json or PASCAL so it will error out if
+those are not provided.)
+6. Add python dependencies to the install script.
+
+## Quick Start Guide
+
+1. Process the Data:
+
+```bash
+rosrun mil_mlp process_data.py --json_path='json_files/JSON_NAME.json --labelmap_path='../docker_tf/transfer_learning/data/LABELMAPNAME.pbtxt
```

Comment:
Still would prefer a less ambiguous name for this script. Why not `convert_labels_to_tfrecord`?